docs/blog/field-types.md

```diff
@@ -44,17 +44,17 @@ Although technically similar to the `Text` field, conceptually, `Markdown` repre
 This completes the basic criteria of how we decide on new field types over config options or project specific customsation:
 
 1. Is data in the Database or GraphQL a different type?
-1. Is it conceptually a different field type for users?
-1. Is the UI incompatible with other values using the same type?
+2. Is it conceptually a different field type for users?
```

Comment:
1. 
1.
1. 

Is the correct format for ordered lists in markdown. It's done this way so you never need to change the numbers when you reorder.

Reply:
You'll have to take it up with `yarn format`, I'm just doing whatever it does 🤷‍♂️

Reply:
(Which is to say I agree with you, but if we're going to go with that pattern, we need to update our tooling to make it the same everywhere)

Reply:
It's it's prettier no drama